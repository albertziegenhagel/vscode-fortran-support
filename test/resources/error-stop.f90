error stop
error stop 10
error stop "val"//"10"
error stop label
