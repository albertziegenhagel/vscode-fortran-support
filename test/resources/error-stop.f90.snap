>error stop
#^^^^^^^^^^ source.fortran.free meta.statement.control.errorstop.fortran keyword.control.errorstop.fortran
>error stop 10
#^^^^^^^^^^ source.fortran.free meta.statement.control.errorstop.fortran keyword.control.errorstop.fortran
#          ^ source.fortran.free meta.statement.control.errorstop.fortran
#           ^^ source.fortran.free meta.statement.control.errorstop.fortran constant.numeric.fortran
>error stop "val"//"10"
#^^^^^^^^^^ source.fortran.free meta.statement.control.errorstop.fortran keyword.control.errorstop.fortran
#          ^ source.fortran.free meta.statement.control.errorstop.fortran
#           ^ source.fortran.free meta.statement.control.errorstop.fortran string.quoted.double.fortran punctuation.definition.string.begin.fortran
#            ^^^ source.fortran.free meta.statement.control.errorstop.fortran string.quoted.double.fortran
#               ^ source.fortran.free meta.statement.control.errorstop.fortran string.quoted.double.fortran punctuation.definition.string.end.fortran
#                ^^ source.fortran.free meta.statement.control.errorstop.fortran keyword.other.concatination.fortran
#                  ^ source.fortran.free meta.statement.control.errorstop.fortran string.quoted.double.fortran punctuation.definition.string.begin.fortran
#                   ^^ source.fortran.free meta.statement.control.errorstop.fortran string.quoted.double.fortran
#                     ^ source.fortran.free meta.statement.control.errorstop.fortran string.quoted.double.fortran punctuation.definition.string.end.fortran
>error stop label
#^^^^^^^^^^ source.fortran.free meta.statement.control.errorstop.fortran keyword.control.errorstop.fortran
#          ^ source.fortran.free meta.statement.control.errorstop.fortran
#           ^^^^^ source.fortran.free meta.statement.control.errorstop.fortran meta.parameter.fortran
>